### PR TITLE
feat: Add configuration for currencies supported on particular chain

### PR DESCRIPTION
**Pull request by ameten**
### Description

Add configuration for currencies supported on particular chain. The configuration allows to specify mapping from the identifier of crypto currency which is used in transaction object to UUID which uniquely identify this crypto currency for Hyperlane agent.

### Related issues

- Contributes to https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/4641

### Backward compatibility

Yes

### Testing

Generated agent config (`mainnet-config.json`) and started Scraper with it just to check that new field does not break it.

### CODE REVIEW

File: chains/metadata.yaml [hunk 1]
```diff
@@ -2435,6 +2435,17 @@ neutron:
     - http: https://rpc-arch.mainnet.neutron.tm.p2p.org
     - http: https://rpc-kralum.neutron-1.neutron.org
   slip44: 118
+  supportedCurrencies:
```
Comment (daniel-savu) [inline chains/metadata.yaml:2438]: I'm torn on the naming, let's discuss in the notion doc

File: chains/neutron/metadata.yaml [hunk 2]
```diff
@@ -34,3 +34,14 @@ rpcUrls:
 slip44: 118
 transactionOverrides:
   gasPrice: "0.0075"
+supportedCurrencies:
+  - symbol: TIA
+    description: "It is not TIA on Celestia chain. It is TIA bridged to Neutron chain."
+    identifier: ibc/773B4D0A3CD667B2275D5A4A7A2F0909C0BA0F4059C0B9181E680DDF4965DCC7
```
Comment (jmrossy) [inline chains/neutron/metadata.yaml:40]: `addressOrDenom` may be more consistent with other field names

File: chains/neutron/metadata.yaml [hunk 3]
```diff
@@ -34,3 +34,14 @@ rpcUrls:
...
+    uuid: 0b03525c-ec50-4176-9318-b70636e0c054
```
Comment (jmrossy) [inline chains/neutron/metadata.yaml:41]: Is this just randomly generated when you put together this config?

File: chains/neutron/metadata.yaml [hunk 4]
```diff
@@ -34,3 +34,14 @@ rpcUrls:
 slip44: 118
 transactionOverrides:
   gasPrice: "0.0075"
+supportedCurrencies:
```
Comment (jmrossy) [inline chains/neutron/metadata.yaml:37]: There's already a `nativeToken` field. I wonder if this should be folded into that? Or at least named `supportedTokens` or something or consistency?

Comment (jmrossy) [reply chains/neutron/metadata.yaml:37]: Or `supportedGasTokens`

**Pull request closed without merging**